src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.Native.cs

```diff
@@ -13,18 +13,19 @@ partial class ScrollContentPresenter : IFrameworkTemplatePoolAware
 
 		private object RealContent => Native?.Content;
 
-		public void OnTemplateRecycled()
+		public void OnTemplateRecycled() // fixme@xy: review
```

Comment:
create issue to update template recycling code that interact with tp
and leave a proper comment in code